launcher/StringUtils.cpp

```diff
@@ -212,3 +212,25 @@ QPair<QString, QString> StringUtils::splitFirst(const QString& s, const QRegular
     right = s.mid(end);
     return qMakePair(left, right);
 }
+
+QString StringUtils::htmlListPatch(QString htmlStr)
+{
+    QRegularExpression match("<w*/w*ulw*>|<w*ulw*/w*>");
```

Comment:
Just move this one outside the function and give it a better name, it can remain in this cpp file but just define it as a global variable.
This is not a mandatory one but should improve the speed of the function as the regex will not be reconstructed each time the function is called.

Reply:
you can also do static QRegularExpression

Reply:
![image](https://github.com/PrismLauncher/PrismLauncher/assets/57493648/335c678f-f1f2-4dec-8bad-d6a95ebd084c)
```suggestion
    QRegularExpression match("<\\s/\\s*ul\\s*>");
```
I don't think <ul/> is a valid *closing* tagg so i don't see the need to account for it